File: C++_Programming/Practice/program10.cpp
#include<iostream>
using namespace std;

void CheckEvenOdd(int iNo)
{
    int iRemainder = 0;
    iRemainder = iNo % 2;

    if(iRemainder == 0)
    {
        cout<<"The Number is Even";
    }
    else
    {
        cout<<"The Number is Odd";
    }
}

int main()
{   
    int iValue = 0;
    int iRemainder = 0;

    cout<<"Enter Number : ";
    cin>>iValue;

    CheckEvenOdd(iValue);
    return 0;
}

File: Java_Programming/Practice/program10.java
import java.util.Scanner;

class NumberX
{
    public static void CheckEvenOdd(int iNo)
    {
        int iRemainder = 0;

        iRemainder = iNo % 2;

        if(iRemainder == 0)
        {
            System.out.println("The Numbe is Even ");
        }
        else
        {
            System.out.println("The Numbe is Odd");
        }
    }
}

class program10
{
    public static void main(String A[]) 
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;

        System.out.println("Enter Number : ");
        iValue = sobj.nextInt();

        NumberX.CheckEvenOdd(iValue);
    }
}